### ISSUE TITLE
There is no tutorial

### ISSUE THREAD

**Issue by RankoR** [sshipman/RhombusLib#2]
Can you please write a small tutorial about how to use this library?
Thanks.

**Comment by vikasvmane** [comment]
Yeah,it will be good to have a detailed documentation for how to use the library.Only then it will be easier to implement.sshipman please help.Thanks

**Comment by davalidator** [comment]
Is there a small tutorial for how to use the library?
I tried implementing it as such:

In my main activity, I created an instance of AudioMonitor with a handler that can handle the messages of MessageType.
When the message is of type MessageType.DATA, I used an AudioDecoder instance to process the data and display the resulting string.

I believe all of this is fine.

I am confused on how to properly start and stop the audio monitoring.
Should the monitoring happen in a background or maybe in a service?
Please advise.

**Comment by davalidator** [comment]
Never mind. I was able to figure it out by using a Thread.

**Comment by cdunworth** [comment]
a small tutorial/demo app would be very helpful, specially for those new to Android & Java
very interesting work
Thank you

**Comment by ozarka30** [comment]
davalidator, if possible do you think you can send me a link to your source code for how you implemented this?  I am working on a project and can't figure out how to do this.  I'm also new to java and android programming which might be why?

**Comment by davalidator** [comment]
chunworth and ozarka30,

I uploaded an example app with my modifications to the RhombusLib. This should be a good start.
https://github.com/davalidator/rhombuslibexample

**Comment by ozarka30** [comment]
Thank you for sharing it helped a lot!

If possible do you think you can take a look at this for me?  I'm having some trouble getting the final message to output to a WebView.

This is my github if you have some time.....
https://github.com/GhostdeLegend/Club-Points-Web-App

Thank you for the help again.

**Comment by AmitaInsan** [comment]
Hello i am using this library to read Magnetic strips using the given demo https://github.com/GhostdeLegend/Club-Points-Web-App in this link. I want to get Credit Card company  Name also ,like in Rhombus app is dislaying . Please share the code for this.

Thanks in Advance :)

**Comment by cdunworth** [comment]
Hi,
At highest level
  numbers starting    3 are issued Diners or Amex
                                 4 are issued by VISA
                                 5 are issued MASTERCARD
                                 6 might be MAESTRO
There are of course numerous exceptions to this norm
There are some WIKIS with this info but you need use something like
http://www.binlist.net/

hope this helps

**Comment by AmitaInsan** [comment]
Hello, i am using the given sample app for getting magnetic strips on credit card by this link  https://github.com/davalidator/rhombuslibexample using Rhombus library .But sample app is not displaying Credit Card name .Please share the code to get the Name of credit card as Rhombus app is displaying . 
 Thanks in advance.
